### ISSUE TITLE
Validate normalization

### ISSUE THREAD

**Issue by stuartcampbell** [NSLS-II/PyXRF#157]
Input known data set and check we get the expected result

**Comment by stuartcampbell** [comment]
This should probably also migrate to become an automated test

**Comment by dmgav** [comment]
Currently normalization is performed as follows:
```
                if self.scaler_data is not None:
                    if k in self.name_not_scalable:
                        data_dict = self.dict_to_plot[k]
                    else:
                        data_dict = self.dict_to_plot[k]/self.scaler_data * np.mean(self.scaler_data)
                else:
                    data_dict = self.dict_to_plot[k]
```
(pyxrf/model/draw_image.py, 375-381) 
where ```dict_to_plot``` contains data sets that need to be normalized by ```self.scaler_data```. ```dict_to_plot``` may contain ```self.scaler_data```: in this case it will be normalized by itself (constant). Care is taken that ```self.scaler_data``` does not contain zeros. Andy requested that normalization should be changed as follows:
```
                        data_dict = self.dict_to_plot[k]/self.scaler_data
```
at least for normalization by ```i0``` (beam intensity readings from ion chamber) and ```time_diff``` (duration of measurement period), since data will still carry undistorted quantitative information. Other normalizations don't have too much physical meaning and may remain as they are.

**Comment by dmgav** [comment]
The issue is addressed in pull request #165
